app/services/random_phrase.rb

```diff
@@ -0,0 +1,19 @@
+class RandomPhrase
+  attr_reader :words
```

Comment:
If length is the important metric (rather than number of words), I wouldn't myself be concerned with testing the number of words